src/vos/vos_common.c

```diff
@@ -125,7 +128,7 @@ vos_profile_start(char *path, int avg)
 void
 vos_profile_stop()
 {
-	struct vos_tls *tls = vos_tls_get();
+	struct vos_tls *tls = vos_tls_get(true);
```

Comment:
Minor: vos_profile_start/stop() is dead code, they could be removed.

Reply:
Sure.